ui/src/views/TableCreate.vue

```diff
@@ -60,13 +60,17 @@ export default Vue.extend({
       const { representation } = await expects.load()
       const { root: shape } = representation
       if (shape) {
+        const sourceProperty = shape.property.find(p => p.class?.equals(ns.cc.CSVSource))
+        sourceProperty[ns.hashi.collection.value] = this.sourcesCollection
+
```

Comment:
This part is where the UI now cuts corner a little because there is currently no good way to find the collection URL only by the `sh:class` annotation